.travis.yml

```diff
@@ -0,0 +1,4 @@
+language: objective-c
+xcode_project: loopr-ios.xcodeproj # path to your xcodeproj folder
+# xcode_scheme: loopr-iosUITests
```

Comment:
We just updated the schemes to support configurations for UP wallet, vivwallet and loopr  two days ago. loopr-iosUITests was deprecated. Let me add it back.